clang/lib/CodeGen/CGCall.cpp

```diff
@@ -1846,8 +1847,18 @@ static llvm::fp::FPAccuracy convertFPAccuracy(StringRef FPAccuracyStr) {
       .Case("cuda", llvm::fp::FPAccuracy::CUDA);
 }
 
+static int32_t convertFPAccuracyToAspect(StringRef FPAccuracyStr) {
```

Comment:
Do we need to add an assert here to ensure this function is called with appropriate FPAccuracyStr?

Thanks

Reply:
Added assert, thanks.